R/Flag.R

```diff
@@ -73,6 +77,28 @@ Flag <- function(
     as.character() %>%
     as.numeric() # Parse from factor to numeric
 
+  #filter to site with enough observations via AccrualThreshold and AccrualMetric
+  if (!is.null(nAccrualThreshold) && !is.null(strAccrualMetric)) {
+    if(strAccrualMetric == "Denominator"){
+      Accrual_Flag <- dfFlagged$Denominator < nAccrualThreshold
+    } else if(strAccrualMetric == "Numerator"){
+      Accrual_Flag <-  dfFlagged$Numerator < nAccrualThreshold
+    } else if(strAccrualMetric == "Difference"){
+      Accrual_Flag <- (dfFlagged$Denominator - dfFlagged$Numerator) < nAccrualThreshold
+    }
+    dfFlagged$Score[Accrual_Flag] <- NA
+    dfFlagged$Flag[Accrual_Flag] <- NA
+
+    LogMessage(
+      level = "info",
+      message = paste0(
+        filtered_sites <- sum(Accrual_Flag),
+        " Group(s) have insufficient sample size due to KRI denominator less than {nAccrualThreshold}. \nThese group(s) will not have KRI score and flag summarized."
+      ),
+      cli_detail = "alert_info"
+    )
+  }
+
```

Comment:
would it make sense to carve this out into it's own function? Would make unit testing a bit cleaner I think ...